eoscompanion/routes.py

```diff
@@ -1402,8 +1402,11 @@ def _on_got_wrapped_stream(error, result):
                 response_headers.replace('Connection', 'keep-alive')
 
                 # Add the article thumbnail uri to the header
+                # we only want to add the image when it is content
+                # from a Wikipedia or Wikihow source
                 thumbnail_uri = content_metadata.get('thumbnail', None)
-                if thumbnail_uri is not None:
+                is_wiki_source = content_metadata.get('source', None) in ('wikipedia', 'wikihow')
+                if thumbnail_uri is not None and is_wiki_source:
```

Comment:
Is this also true of other properties we will add to the headers? If so, perhaps we want to break all this code out into a separate function.

Reply:
The title we have already on the client side, we do not need to send this for now. Not sure about the other cases. I would say we refactor when we get there.

Reply:
OK fair enough. +1 from me then.